bears/rest/reSTLintBear.py

```diff
@@ -14,13 +28,17 @@ class reSTLintBear(LocalBear):
     LICENSE = 'AGPL-3.0'
     CAN_DETECT = {'Formatting', 'Syntax'}
 
-    def run(self, filename, file):
+    def run(self, filename, file, ignore_unknown_roles: bool=False):
```

Comment:
c.f. https://github.com/coala/coala-bears/issues/1723

Also see https://github.com/coala/coala/issues/3044#issuecomment-261856917 , where I think I saw extra error messages from restlint than rstcheck emitted .

I think you might be right about the need for a purer rst linter.  https://pypi.python.org/pypi/restructuredtext_lint reminds me ... we need a rst-checker which validates README.rst so that it can be used for PyPI, which has stricter rules than Sphinx, and stricter rules than GitHub's RST parser.

*Maybe* we can request rstcheck has a strict rst mode.  Oddly enough, [this coala project](http://projects.coala.io/#/projects?project=improve_sphinx_support_in_rstcheck) seemed like it might have created a more clear separation between docutils mode and sphinx mode.

And if we need a pure rst linter, then we dont want to allow it to validate sphinx rst.  It should fail! ;-)
And we should implement restLintBear on any README.rst or other file which is included in the packaging for PyPI, and not other RST files.